language/setup.py

```diff
@@ -61,7 +61,7 @@
 
 setup(
     name='google-cloud-language',
-    version='0.25.0',
+    version='0.26.0',
```

Comment:
@tseaver This means the uber-package will need a release

@lukesneeringer Can you confirm?